How_to_guides/Analyse_multiple_polygons.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Hi, these comments seem to be from the old notebook : &nbsp;&nbsp;</p><p> </p><p># Append results to a dictionary using the attribute</p><p>&nbsp;&nbsp;# column as an key</p><p>&nbsp;&nbsp;# results.update({str(row[attribute_col]): ds_masked})</p><p> </p><p>Suggest updating to reflect new usage?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>